lms/static/sass/bootstrap/_components.scss

```diff
@@ -23,3 +23,10 @@
     margin-left: $baseline;
   }
 }
+
+.alert.alert-info a {
+  // Need to have sufficient contrast for blue links on blue background
+  // Based this on some existing css for this use case
+  color: #245269;
+  font-weight: bold;
+}
```

Comment:
this makes the course home banner link looks the same as the current appearance of the in-course banner link
<img width="717" alt="screen shot 2018-12-03 at 12 52 56 pm" src="https://user-images.githubusercontent.com/5958221/49391904-f788e180-f6fa-11e8-8e62-1fdb71b2e8bb.png">

Reply:
@MatthewPiatetsky - given that this is just making the banner consistent with the banner elsewhere in the course 👍. We will want to take a more holistic look at this banner post release. FYI: @rrakhshan